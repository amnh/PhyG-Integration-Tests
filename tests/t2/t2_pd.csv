Alentus,Amblypygid,Americhernus,Aportus,Artemia,Centruoides,Chanbria,Gea,Hadrurus,Hypochilus,Limulus,Mastigoproctus,Pauroctonus,Rhiphicephalus,Thelichoris,Thermobius,Vonones
"0.0","170.0","267.0","121.0","217.0","197.0","183.0","241.0","187.0","184.0","183.0","184.0","200.0","201.0","186.0","186.0","170.0"
"170.0","0.0","229.1075050709939","180.35928143712576","190.90140845070422","140.80487804878047","138.48275862068965","202.63286004056795","177.53658536585365","140.80487804878047","127.54065040650406","135.42799188640973","154.0691056910569","169.36161616161618","125.5","175.7","132.10526315789474"
"267.0","229.1075050709939","0.0","256.5109780439122","259.5855130784708","251.50912778904663","258.63691683569976","268.8194726166328","251.50912778904663","239.29006085192697","245.3995943204868","252.52738336713995","254.56389452332655","261.6484848484849","237.25354969574036","266.06","234.7408906882591"
"121.0","180.35928143712576","256.5109780439122","0.0","226.45109780439122","196.39121756487026","192.38323353293413","237.47305389221557","204.40718562874252","220.43912175648703","187.37325349301398","189.37724550898204","200.3992015968064","219.437125748503","212.4231536926148","195.38922155688624","167.33333333333334"
"217.0","190.90140845070422","259.5855130784708","226.45109780439122","0.0","130.29778672032194","159.58953722334005","189.89134808853117","185.85110663983903","163.6297786720322","161.60965794768612","164.63983903420524","187.8712273641851","193.93158953722332","163.6297786720322","185.74","183.83098591549296"
"197.0","140.80487804878047","251.50912778904663","196.39121756487026","130.29778672032194","0.0","143.57403651115618","168.01217038539554","126.52032520325203","128.5609756097561","127.80040733197556","133.39148073022312","132.91242362525458","171.389898989899","135.7032520325203","181.724","148.3643724696356"
"183.0","138.48275862068965","258.63691683569976","192.38323353293413","159.58953722334005","143.57403651115618","0.0","173.10344827586206","188.37728194726165","135.42799188640973","105.89858012170384","109.97160243407707","154.77484787018255","160.23434343434346","124.22718052738337","180.72","130.07287449392712"
"241.0","202.63286004056795","268.8194726166328","237.47305389221557","189.89134808853117","168.01217038539554","173.10344827586206","0.0","215.87018255578093","152.73833671399595","174.12170385395538","177.17647058823528","188.37728194726165","225.13939393939395","172.08519269776875","220.88","189.01214574898785"
"187.0","177.53658536585365","251.50912778904663","204.40718562874252","185.85110663983903","126.52032520325203","188.37728194726165","215.87018255578093","0.0","164.27235772357722","164.27235772357722","177.17647058823528","130.60162601626016","198.7717171717172","182.6382113821138","192.768","179.8663967611336"
"184.0","140.80487804878047","239.29006085192697","220.43912175648703","163.6297786720322","128.5609756097561","135.42799188640973","152.73833671399595","164.27235772357722","0.0","123.45934959349593","122.19066937119675","169.3739837398374","172.40404040404042","85.70731707317073","185.74","148.3643724696356"
"183.0","127.54065040650406","245.3995943204868","187.37325349301398","161.60965794768612","127.80040733197556","105.89858012170384","174.12170385395538","164.27235772357722","123.45934959349593","0.0","117.09939148073022","135.979633401222","163.2767676767677","133.66260162601625","156.624","112.79757085020242"
"184.0","135.42799188640973","252.52738336713995","189.37724550898204","164.63983903420524","133.39148073022312","109.97160243407707","177.17647058823528","177.17647058823528","122.19066937119675","117.09939148073022","0.0","132.3732251521298","158.20606060606062","126.26369168356997","157.62800000000001","130.07287449392712"
"200.0","154.0691056910569","254.56389452332655","200.3992015968064","187.8712273641851","132.91242362525458","154.77484787018255","188.37728194726165","130.60162601626016","169.3739837398374","135.979633401222","132.3732251521298","0.0","172.40404040404042","147.9471544715447","159.636","138.20242914979755"
"201.0","169.36161616161618","261.6484848484849","219.437125748503","193.93158953722332","171.389898989899","160.23434343434346","225.13939393939395","198.7717171717172","172.40404040404042","163.2767676767677","158.20606060606062","172.40404040404042","0.0","101.41414141414143","176.704","159.22020202020204"
"186.0","125.5","237.25354969574036","212.4231536926148","163.6297786720322","135.7032520325203","124.22718052738337","172.08519269776875","182.6382113821138","85.70731707317073","133.66260162601625","126.26369168356997","147.9471544715447","101.41414141414143","0.0","178.712","144.2995951417004"
"186.0","175.7","266.06","195.38922155688624","185.74","181.724","180.72","220.88","192.768","185.74","156.624","157.62800000000001","159.636","176.704","178.712","0.0","158.632"
"170.0","132.10526315789474","234.7408906882591","167.33333333333334","183.83098591549296","148.3643724696356","130.07287449392712","189.01214574898785","179.8663967611336","148.3643724696356","112.79757085020242","130.07287449392712","138.20242914979755","159.22020202020204","144.2995951417004","158.632","0.0"
