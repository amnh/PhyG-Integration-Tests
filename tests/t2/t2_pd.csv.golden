Alentus,Amblypygid,Americhernus,Aportus,Artemia,Centruoides,Chanbria,Gea,Hadrurus,Hypochilus,Limulus,Mastigoproctus,Pauroctonus,Rhiphicephalus,Thelichoris,Thermobius,Vonones
"0.0","170.0","267.0","121.0","217.0","197.0","183.0","241.0","187.0","184.0","183.0","184.0","200.0","201.0","186.0","186.0","170.0"
"170.0","0.0","225.0","180.0","189.0","138.0","136.0","199.0","174.0","138.0","125.0","133.0","151.0","167.0","123.0","175.0","130.0"
"267.0","225.0","0.0","256.0","257.0","247.0","254.0","264.0","247.0","235.0","241.0","248.0","250.0","258.0","233.0","265.0","231.0"
"121.0","180.0","256.0","0.0","226.0","196.0","192.0","237.0","204.0","220.0","187.0","189.0","200.0","219.0","212.0","195.0","167.0"
"217.0","189.0","257.0","226.0","0.0","129.0","158.0","188.0","184.0","162.0","160.0","163.0","186.0","192.0","162.0","185.0","182.0"
"197.0","138.0","247.0","196.0","129.0","0.0","141.0","165.0","124.0","126.0","125.0","131.0","130.0","169.0","133.0","181.0","146.0"
"183.0","136.0","254.0","192.0","158.0","141.0","0.0","170.0","185.0","133.0","104.0","108.0","152.0","158.0","122.0","180.0","128.0"
"241.0","199.0","264.0","237.0","188.0","165.0","170.0","0.0","212.0","150.0","171.0","174.0","185.0","222.0","169.0","220.0","186.0"
"187.0","174.0","247.0","204.0","184.0","124.0","185.0","212.0","0.0","161.0","161.0","174.0","128.0","196.0","179.0","192.0","177.0"
"184.0","138.0","235.0","220.0","162.0","126.0","133.0","150.0","161.0","0.0","121.0","120.0","166.0","170.0","84.0","185.0","146.0"
"183.0","125.0","241.0","187.0","160.0","125.0","104.0","171.0","161.0","121.0","0.0","115.0","133.0","161.0","131.0","156.0","111.0"
"184.0","133.0","248.0","189.0","163.0","131.0","108.0","174.0","174.0","120.0","115.0","0.0","130.0","156.0","124.0","157.0","128.0"
"200.0","151.0","250.0","200.0","186.0","130.0","152.0","185.0","128.0","166.0","133.0","130.0","0.0","170.0","145.0","159.0","136.0"
"201.0","167.0","258.0","219.0","192.0","169.0","158.0","222.0","196.0","170.0","161.0","156.0","170.0","0.0","100.0","176.0","157.0"
"186.0","123.0","233.0","212.0","162.0","133.0","122.0","169.0","179.0","84.0","131.0","124.0","145.0","100.0","0.0","178.0","142.0"
"186.0","175.0","265.0","195.0","185.0","181.0","180.0","220.0","192.0","185.0","156.0","157.0","159.0","176.0","178.0","0.0","158.0"
"170.0","130.0","231.0","167.0","182.0","146.0","128.0","186.0","177.0","146.0","111.0","128.0","136.0","157.0","142.0","158.0","0.0"
